test/e2e/e2e_test.go

```diff
@@ -43,12 +43,12 @@ func init() {
 
 	prfs, _ := c.ListProfiles()
 	if len(prfs) == 0 {
-	        fmt.Println("Start creating profile...")
+		fmt.Println("Start creating profile...")
```

Comment:
I am curious why this line was changed

Reply:
I don't know why, it's probably because of the IDE.